services/coderabbit/coderabbit-stats.service.js

```diff
@@ -0,0 +1,69 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  reviews: Joi.number().required(),
+}).required()
+
+class CodeRabbitStats extends BaseJsonService {
+  static category = 'analysis'
+  static route = {
+    base: 'coderabbit',
+    pattern: 'stats/:provider/:org/:repo',
+  }
+
+  static examples = [
+    {
+      title: 'CodeRabbit Review Stats',
+      namedParams: {
+        provider: 'github',
+        org: 'coderabbitai',
+        repo: 'ast-grep-essentials',
+      },
+      staticPreview: this.render({
+        reviews: 101,
+      }),
+      documentation: 'Shows the number of CodeRabbit reviews for a repository',
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'CodeRabbit',
+    labelColor: '171717',
+  }
+
+  static render({ reviews }) {
+    return {
+      message: `${reviews} Reviews`,
```

Comment:
From

https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines

> The left-hand side of a badge should not advertise. It should be a lowercase noun succinctly describing the meaning of the right-hand side.

So rather than `CodeRabbit | 7 Reviews`, this badge should be either `reviews | 7` or `coderabbit reviews | 7`

Reply:
I have updated the text accordingly.

Reply:
The label text should be lowercase, even where we're including a service name

Reply:
Made this change.